# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../junit/build_destination.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./build.xml	2005-07-10 21:46:18.409753920 +0200
+++ ./build.xml	2005-07-10 22:16:45.360015264 +0200
@@ -3,7 +3,7 @@
 	<property file="${user.home}/.junit.properties" />
 	<property name="build.compiler" value="classic" />
 	<property name="version" value="3.8.1" />
-	<property name="dist"  value="junit${version}" />
+	<property name="dist"  value="dist" />
 	<property name="versionfile"  value="junit/runner/Version.java" />
 	<property name="zipfile"  value="${dist}.zip" />
 	<target name="init">
@@ -32,11 +32,6 @@
 		<delete dir="${dist}" />
 		<mkdir dir="${dist}" />
 		<jar 
-			jarfile="${dist}/src.jar"
-			basedir="."
-			excludes="**/*.jar, junit/tests/**, junit/samples/**, **/*.class, doc/**, README.html, .classpath, .project, cpl-v10.html"
-		/>
-		<jar 
 			jarfile="${dist}/junit.jar"
 			basedir="."
 			excludes="**/*.jar, junit/tests/**, junit/samples/**, **/*.java, doc/**, README.html,  build.xml, jar-manifest.txt, .classpath, .project, cpl-v10.html"
@@ -55,18 +50,18 @@
 		     includes="junit/tests/runner/LoadedFromJar.class"
 		 /> 
 
-		<mkdir dir="${dist}/javadoc" />
+		<mkdir dir="${dist}/docs/javadoc" />
 		<javadoc 
 			sourcepath="."
 			packagenames="junit.framework.*, junit.extensions.*"
-			destdir="${dist}/javadoc"
+			destdir="${dist}/docs/javadoc"
 			author="false"
 			version="false"
 			use="false"
 			windowtitle="JUnit API"
 			stylesheetfile="stylesheet.css"
 		/>
-		<copy todir="${dist}/doc">
+		<copy todir="${dist}/docs">
 			<fileset dir="doc"/>
 		</copy>		
 		<copy file="README.html" tofile="${dist}/README.html" />
